### DISCUSSION TITLE
Rust project being classified as RenderScript

### DISCUSSION THREAD

**Discussion by RonaldColyar** [github-linguist/linguist#5702] [Classification] [answered]
This repository contains a Rust project that doesn't have any other languages associated with it, yet it is being classified as RenderScript. 
![issue](https://user-images.githubusercontent.com/35206353/147203559-fc2992c8-ef53-4357-997b-18f9d00d495d.png)

Repository Link: https://github.com/Collaborative-IoT/GeneralControlServer

**Comment by lildude** [comment, marked as answer]
This is because of the file at https://github.com/Collaborative-IoT/Merlin/blob/master/src/communication/communication_handler.rs

It only contains a comment and doesn't match any of the [heuristics](https://github.com/github/linguist/blob/80f3531e8a1014a23f4606458e5a528053ed3cac/lib/linguist/heuristics.yml#L503-L510) so falls through to the classifier which has made a guess based on the samples we've got for both. In this case, I suspect it's the `/*` which occurs in our RenderScript sample but not our Rust samples which has caused this - our only Rust sample with a `/* */` style comment starts with `/*!`.

You should find this file will be correctly identified when you start adding code to it. You can also change the `/*` to `/*!` or implement an override to force it in the mean time.

**Reply by RonaldColyar** [reply]
Thanks, sorry for the link change.
